packages/core/src/lib/LinkTo/LinkTo.styles.ts

```diff
@@ -7,32 +7,39 @@ import type { LinkToProps } from './types';
 
 // styles
 type TStyleProps = Pick<LinkToProps, 'iconLayout'>;
+interface LinkStyledProps {
+    underline: boolean;
```

Comment:
could you please rename this to `isUnderline` ?
Boolean props should have this type of naming convention so it is easier to understand while using the component 😄

Reply:
Thanks for the feedback 🙌
...renamed it 😀